server/api/user/user.controller.js

```diff
@@ -340,3 +347,43 @@ exports.removeTech = function(req,res){
         }
     });
 };
+
+/**
+ * Set reset token for user and email it the password token will expire after 24 hours.
+ */
+exports.resetPassword = function(req, res){
+    var userEmail = req.body.email;
+    User.findOne({
+        email: userEmail.toLowerCase()
+    }, function (err, user){
+        if (err) return res.json(401, err);
+        if (!user) return res.json(401, {message: 'User does not exist'});
```

Comment:
Should not let this information escape to the client. Make every email have a check email option. We should also remove the email is not registered and make it a generic 'email/password incorrect'